### PR TITLE
Look for Lapack functions in OpenBLAS

### PR DESCRIPTION
**Pull request by jeanlucf22**

### CODE REVIEW

**Review by suemni — APPROVED** [review]

File: CMakeLists.txt [hunk 1]
```diff
@@ -460,7 +460,9 @@ if(BLAS_FOUND)
     -DC_ZAXPY=${C_ZAXPY})
 endif()
 
-if(LAPACK_FOUND)
+#check for functions in Lapack
+#(or OpenBLAS since OpenBLAS typically contains Lapack)
```
Comment (nicolasbock) [inline CMakeLists.txt:464]: Why is this change necessary? In my local tests LAPACK is correctly detected with OpenBLAS.

Comment (jeanlucf22) [reply CMakeLists.txt:464]: On Summit, it is not:

-- BLA_VENDOR=OpenBLAS
-- Looking for Fortran sgemm
-- Looking for Fortran sgemm - found
-- Found BLAS: /sw/summit/spack-envs/base/opt/linux-rhel8-ppc64le/gcc-9.3.0/openblas-0.3.17-xre5bzetoh5rctek6hyif6bt4wt2h7c2/lib/libopen                blas.so
-- BLAS_LIBRARIES=/sw/summit/spack-envs/base/opt/linux-rhel8-ppc64le/gcc-9.3.0/openblas-0.3.17-xre5bzetoh5rctek6hyif6bt4wt2h7c2/lib/libo                penblas.so
-- LAPACK_LIBRARIES=

Comment (nicolasbock) [reply CMakeLists.txt:464]: Interesting. What version `CMake` is running there?

Comment (jeanlucf22) [reply CMakeLists.txt:464]: cmake version 3.23.2

**Review by cnegre — APPROVED** [review]
